### ISSUE TITLE
Ability to disable processing messages and events in muted rooms and load them only on demand

### ISSUE THREAD

**Issue by MurzNN** [vector-im/riot-web#7438]
Many Riot users have joined to large active rooms (like Matrix HQ) in `Mute` mode, because they need to talk in them very rarely, but sometimes want to look few last messages, so every time leave and join that rooms is not good way.

At now Riot receives all messages from all rooms, even from muted rooms, so it spend unnecessary memory and cpu time for processing those messages.

Will be good to have option to disable processing muted messages in local Riot instance - not show notifications and not add them into Riot app memory. And load event history on those rooms on demand, only if user manually open this room.

Even if server will continue sending it to client - we can ignore processing if detect that this message is for muted room.

**Comment by MurzNN** [comment]
Seems we can implement filtering of muted rooms in `/sync` action via filling `not_rooms` string using [RoomFilter](https://matrix.org/docs/spec/client_server/r0.5.0#filtering), and sync them only via per-room basis on demand (when user opens needed room)?
For not broke previous behavior, this mode can be optionally, included in some `Power save` group.

**Comment by progserega** [comment]
Yes - it is nice behavior: skip sync for already bold rooms. Bold - tell user that muted room have new unread messages, but if user not want read this room - it is skip from sync. And sync this room, if user enter to this "bold room".
